README.adoc

```diff
@@ -125,10 +121,12 @@ Next, create a `src/main/liberty/config/server.xml` file:
 include::finish/src/main/liberty/config/server.xml[tags=server]
 ----
 
-The `mpMetrics-1.0` feature provides the `https://localhost:9443/metrics` endpoint.
+The `mpMetrics-1.0` feature enables MicroProfile Metrics support in Open Liberty. Note that this
+feature requires SSL and the configuration has been provided to you.
 
-The `quickStartSecurity` and `keyStore` configuration elements provide basic security to secure the endpoint. When you visit
-the endpoint, you use these credentials to login in order to view the metrics.
+The `quickStartSecurity` and `keyStore` configuration elements provide basic security to secure the
+server. When you visit the `/metrics` endpoint, you use the credentials defined in the server
+configuration to login to view the data.
```

Comment:
I would remove the second instance of "you" and change "login" to two words because it's a verb here:
"When you visit the `/metrics` endpoint, use the credentials defined in the server configuration to log in to view the data."